### PR TITLE
docs: refactor pluggable auth docs for clarity and editing.

### PR DESCRIPTION
**Pull request by kevin-dimichel**
## What changes are proposed in this pull request?

Pluggable Authentication is a major addition to FiftyOne Teams.  Having not been involved in the project, I figured I would contribute to the docs (in the spirit of improvement)

## How is this patch tested? If it is not, please explain why.

n/a docs only changes.

### CODE REVIEW

File: docs/source/teams/pluggable_auth.md [hunk 1]
```diff
@@ -2,84 +2,80 @@
...
 
-This is where you configure the deployment wide configuration of FiftyOne
-Teams. When logging into FiftyOne Teams itself as an admin you are in the
-context of an organization, and settings there only apply to that organization.
-On the other hand, the Super Admin UI allows you to administer all
-organizations, and global configuration such as Identity Providers, Session
-timeouts, and JS hooks.
+The Super Admin UI contains FiftyOne Teams deployment wide configurations. When
+logging into FiftyOne Teams as an admin, you are in the context of an
+organization.Settings are scoped by organization and only apply to that
```
Comment (ritch) [inline docs/source/teams/pluggable_auth.md:55]:
```suggestion
organization. Settings are scoped by organization and only apply to that
```

File: docs/source/teams/pluggable_auth.md [hunk 2]
```diff
@@ -124,59 +120,60 @@ API to ensure a comprehensive transfer of information.
...
+1. Linking the user's account to the new authentication provider by creating an
    account reference via `POST /cas/api/accounts`.
 
 The final step in the migration involves changing the `FIFTYONE_AUTH_MODE`
 setting from legacy to internal. This change officially activates Internal
 Mode, completing the migration process.
 -->
 
-## Getting started with internal mode
+## Getting Started with Internal Mode
```
Comment (ritch) [inline docs/source/teams/pluggable_auth.md:132]: The style for our docs is like this:

> Getting started with internal mode

Here's a few examples from a random search:

 - `Selecting samples of interest`
 - `Add predictions to the dataset`
 - `Removing redundant images`

**Review by ritch — CHANGES_REQUESTED** [review]
Minor comments, otherwise LGTM

File: docs/source/teams/pluggable_auth.md (hunk 2 above)
Comment (kevin-dimichel) [reply docs/source/teams/pluggable_auth.md:132]: Seems like we inconsistently capitalize headings. I'll change it back as you request.

**Review by ritch — APPROVED** [review]

**Pull request merged**